python_modules/libraries/dagster-aws/dagster_aws/athena/resources.py

```diff
@@ -23,7 +25,7 @@ class AthenaTimeout(AthenaError):
     pass
 
 
-class AthenaResource:
+class AthenaClient:
```

Comment:
Hmmm is this not a breaking change? `AthenaResource` was public and now methods like `execute_query` don't exist

Reply:
I think this is worth discussing in a call  - I'm not sure what the best path forward is for cases like this to preserve compat while adding a factory wrapper.